src/store.js

```diff
@@ -16,6 +16,9 @@ const readmeContext = require.context('onlyFrontmatter!lessonSrc/', true,
   /^\.\/[^\/]*\/[^\/]*\/README(_[a-z]{2})?\.md$/);
 const lessons = getLessons(lessonContext, readmeContext, courseContext);
 
+// Add all resources, i.e. all files in lessonSrc that does not end in '.md':
+require.context('lessonSrc/', true, /\.(?!md$)[^.]+$/);
```

Comment:
I think perhaps this line is not necessary. It forces webpack to include ALL files that don't end in `.md`, but I think markdown-it-modify-token (setting data-src) combined with the new `attrs`-parameter for html-loader should include/require all resources that are linked to from the lessons. So by deleting this line (require.context) we avoid copying resources that are not linked to (i.e. not used).

Needs to be tested first though.